### PR TITLE
How to Setup the Glassmorphism Frosty Effect in a React.js App with TypeScript

### LINKED ISSUE #5694: How to Setup the Glassmorphism Frosty Effect a React.js App with TypeScript
### Topic Suggestion
Enter topic suggestion here...

#### Pre-submission advice
By following all our pre-submission advice and reviewing our Resources folder, you will maximise your chances of your topic being approved.

We ask that you please be patient as our team works through approving and publishing all articles/tutorials in a timely manner.

Allow 1-3 days for a topic to be reviewed and/or approved - allow 3-7 days for an articles to be reviewed and/or published (subject to vary depending on _volume and/or backlog of articles_).

Be sure to visit our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for tools, resources, and example articles that will help you propose and write a successful article.

**Please ensure that you have only one open issue + linked pull request at a time. This will ensure that we complete the article in a timely manner from inception to publishing.)**

**We tend to stray away or tend not to publish reviews/comparisons of commercial product offerings.**

### Writing sample(s): 
Include any links or writing samples - to help our team better gauge your writing quality.

#### Proposed title of article

How to Setup an Add the Glassmorphism Frosty Effect a React.js Application with TypeScript

#### Proposed article introduction

Glassmorphism is a cool UI design trend in the graphic design world that allows you to create an amazing illusion of glass in your application artwork. This guide will discuss the concept of Glassmorphism and a step by step illustrations on how you can achieve the Glassmorphism effect in a React.js application.

#### Key takeaways 

What are Glassmorphism
Overview
Set up React with CRA
A step by step Glassmorphism implementation using React
Conclusion

#### Article quality

Glassmorphism can be implemented in any application of your choice. This guide will help the reader implement a Glassmorphism effect in React.js. We will create a Handy React application that will help the reader understand and implement Glassmorphism In any React.js app.

#### References
Please list links to any published content/research that you intend to use to support/guide this article.

### Conclusion
Finally, remove the Pre-Submission advice section and all our blockquoted notes as you fill in the form before you submit.
We look forwarding to reviewing your topic suggestion.

### Templates to use as guides
- [How To Guide Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/how-to-guide-template.md)
- [Software Review Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/software-review-template.md)
- [Tutorial Template](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/tutorial-template.md)

### PR DESCRIPTION
**Pull request by mosesreigns**
## Our **Peer Reviewer and Content Moderator teams do NOT provide any revisions services.** All revisions and edits should be completed on your own forked repo (as to not take up room in the queue) in order for our team to review them all in a timely manner. 

**ANY ARTICLE SUBMITTED WITH GLARING ERRORS WILL BE IMMEDIATELY CLOSED.**

>As a rule of thumb - please be sure to only submit articles (pull requests) that are fully polished and ready to be published. Be sure to go through our resources documents for extra and 3rd party (vetted) resources to help improve overall technical writing.

> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [ ] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #5694

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Kayere** [comment]
Plagiarism check done ✅

@mosesreigns - Your article seems to be too short - word count (excluding code snippets) is less than 750. Please consider adding more content to your article to meet the minimum word count.

**Comment by mosesreigns** [comment]
Am on it ASAP @Kayere

**Comment by mosesreigns** [comment]
@adrianmurage Check this out

File: content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md
Comment (adrianmurage) [inline content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md:193]: To what class are these styles being applied?

File: content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md
Comment (adrianmurage) [inline content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md:200]: To what class are these styles being applied?

File: content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md
Comment (adrianmurage) [inline content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md:209]: To what class are these styles being applied?

File: content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md
Comment (adrianmurage) [inline content/articles/setup-glassmorphism-effect-in-reactjs-app/index.md:218]: To what class are these styles being applied?

**Review by adrianmurage — CHANGES_REQUESTED** [review]
@mosesreigns 
Your article is good but it lacks a few things:
1. The article is still shorter than the required word count (excluding code blocks). Kindly think of a way to get it up to par before requesting another review. 
2. The article has a few non-specific styles. Explicitly state to what class are these styles being applied.

**Comment by mosesreigns** [comment]
@adrianmurage @Kayere Regarding the word count, I had capitalized on adding code comments instead of writing them out of the code block. What's your advice on that?

**Comment by adrianmurage** [comment]
> @adrianmurage
> 
> @Kayere
> 
> Should this pull request be closed?

@hectorkambow No need, the necessary changes were made

**Comment by adrianmurage** [comment]
> @adrianmurage @Kayere Regarding the word count, I had capitalized on adding code comments instead of writing them out of the code block. What's your advice on that?

I think that that should be fine. Although for a final word about that you should consult @hectorkambow

**Comment by ahmadmardeni1** [comment]
Great work team. @mosesreigns @adrianmurage

**Comment by hectorkambow** [comment]
How to Setup the Glassmorphism Frosty Effect in a React.js App with TypeScript

**Pull request merged**
